torch/testing/_internal/common_methods_invocations.py

```diff
@@ -17185,12 +17185,19 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
     OpInfo(
         "attn",
         ref=_NOTHING,
-        dtypes=floating_types_and(torch.bfloat16, torch.complex64, torch.complex128),
-        dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16, torch.complex64, torch.complex128),
+        dtypes=floating_types_and(torch.bfloat16),
+        dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
         supports_out=False,
         supports_forward_ad=True,
```

Comment:
In the last PR in the stack, you got forward AD for free because the op was composite, but now you'll have to implement it manually or mark in the OpInfo that they aren't implemented.

Reply:
You should be able to remove the skips in that case